config.go

```diff
@@ -36,7 +36,7 @@ var (
 	ClientVersion = "2022-06-07"
 
 	// Agent version to control agent rollover.
-	AgentVersion = "2022-06-17"
```

Comment:
The agent revision doesn't need to be bumped because the agent executable should already be using the newer route.